pxr/imaging/hgiMetal/blitCmds.mm

```diff
@@ -203,7 +203,11 @@
     
     mtlDesc.mipmapLevelCount = dstTexDesc.mipLevels;
     mtlDesc.arrayLength = dstTexDesc.layerCount;
+#if defined(ARCH_OS_OSX)
     mtlDesc.resourceOptions = MTLResourceStorageModeManaged;
+#else
+    mtlDesc.resourceOptions = MTLResourceStorageModeShared;
```

Comment:
This line seems like it comes up a lot in this PR; it would be nice if we could encapsulate the #if #else in a function, on hgi.mm or capabilities.mm or something.

Reply:
yeah good callout. I'll take a pass at notes next week and hit that up.

Reply:
Should be addressed in the latest commit. I moved it into capabilities as preferredStorageMode, to mirror the existing defaultStorageMode.